packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -268,6 +270,28 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
     }
   }
 
+  public getStartItemIndexInView(): number {
+    const pages = this.state.pages || [];
+    for (const page of pages) {
+      const isPageVisible =
+        page.key &&
+        !page.key.startsWith(SPACER_KEY_PREFIX) &&
+        (this._scrollTop || 0) >= page.top &&
+        (this._scrollTop || 0) <= page.top + page.height;
+      if (isPageVisible) {
+        const startIndexStr = page.key.replace(PAGE_KEY_PREFIX, '');
+        const startIndexInPage = Number(startIndexStr);
+        if (isNaN(startIndexInPage)) {
+          continue;
+        }
+        // Without calculating the exact row height, estimate the row height assuming having same height
```

Comment:
these kind of methods usually take `measureItem?: (itemIndex: number) => number,` parameter in case the user **can** determine the height of an item/page, we should probably allow that here as well

Reply:
added